src/lib/fees/model.ts

```diff
@@ -32,23 +34,12 @@ export const calculateFees: CalculateFeeBps = ({ tradeAmountUsd, foxHeld }) => {
   const midpointUsd = bn(FEE_CURVE_MIDPOINT_USD)
   const feeCurveSteepness = bn(FEE_CURVE_STEEPNESS_K)
 
-  const foxDiscountPercent = BigNumber.minimum(
+  // the following raw values are before the realities of integer bps on-chain
```

Comment:
Can we use a different vernacular to make this easier to grasp? E.g something a la "potential"

Reply:
great point, actioned with jsdoc comment in 19be7d23c47677edafae86e2457a7bc278d0e430